content/contributing/documentation.rst

```diff
@@ -15,407 +15,276 @@ This introductory guide will help you acquire the tools and knowledge you need t
+documentation and then push them to GitHub. Skip this section and go to
+:ref:`contributing/documentation/first-contribution` if you have already completed this step or want
+to make changes from the GitHub interface.
 
-   .. image:: documentation/select-branches-base.png
-
-#. Double-check your :abbr:`PR (Pull Request)` and, when ready, click again on the **Create pull
-   request** button to submit your changes for review by a content writer at Odoo.
-
-   .. image:: documentation/create-pull-request.png
-
-#. You're done! If your changes are approved straight away they will appear in the documentation the
-   very next day. It may also be the case that the reviewer has a question or a remark, so make sure
-   to check your notifications or your emails, depending on your account settings.
-
-.. _contributing/canonical-git-workflow:
-
-Use the canonical Git workflow
-==============================
-
-.. _contributing/prepare-machine:
-
-Prepare your machine
---------------------
-
-.. _contributing/install-git:
-
-Install Git
-~~~~~~~~~~~
-
-We use `Git <https://en.wikipedia.org/wiki/Git>`_ to manage the files of the user documentation.
-It is a tool that allows to track the history of changes made to a file and, more importantly, to
-work on different versions of those files at the same time. It means that you do not need to worry
-about overwriting someone else’s pending work when you start editing the documentation.
-
-You must then configure Git to identify yourself as the author of your future contribution. Enter
-the same email address as the one you used to register on GitHub.
-
-#. Download and install **Git** on your machine.
-#. Verify that `the installation folder of Git is included in your system's PATH variable
-   <win-add-to-path_>`_.
-#. Execute the following commands in a terminal:
+#. .. include:: create_github_account.rst
+#. .. include:: configure_github_account.rst
+#. Go to `github.com/odoo/documentation <https://github.com/odoo/documentation>`_ and click on the
+   :guilabel:`Fork` button in the top right corner to create a fork (:dfn:`your own copy`) of the
+   repository on your account. This creates a copy of the codebase to which you can make changes
+   without affecting the main codebase. Skip this step if you work at Odoo.
+#. .. include:: install_git.rst
+#. .. include:: configure_git_authorship.rst
+#. Clone the sources with Git and navigate into the local repository.
 
    .. code-block:: console
 
-      $ git config --global user.name “Your Name”
-      $ git config --global user.email “youremail@example.com”
-
-.. _contributing/fetch-sources:
+      $ git clone git@github.com:odoo/documentation.git
+      $ cd documentation
 
-Fetch the sources
-~~~~~~~~~~~~~~~~~
+#. Configure Git to push changes to your fork rather than to the main codebase. In the commands
+   below, replace `<your_github_account>` by the name of the GitHub account on which you created the
```

Comment:
```suggestion
   below, replace `<your_github_account>` with the name of the GitHub account on which you created the
```

Reply:
This sounds better to me too, but I need clarification about the rule here. I find resources recommending using "with" when the replacement "has the same function" (e.g., replace a light bulb with a new one) and "by" when the replacement is a different kind (e.g., replace workers by machines). I also find resources recommending "with" when the sentence is passive and "by" otherwise.

Reply:
At the end of your comment, you meant "active," I suppose?
Still trying to figure it out too. I guess I found the same sources as you:
- https://www.btb.termiumplus.gc.ca/tpv2guides/guides/wrtps/index-eng.html?lang=eng&lettr=indx_catlog_r&page=9uHivBTl6Dzk.html
-https://www.quora.com/Whats-the-difference-between-replace-with-and-replace-by)

Reply:
Yes